.vscode/settings.json

```diff
@@ -1,4 +1,3 @@
 {
-    "asciidoc.antora.enableAntoraSupport": true,
```

Comment:
I think that it's probably best to no change this file as part of this PR

Reply:
ah sorry well spotted!